CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# FinOps Open Cost and Usage Specification Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+**Added:**
+
+- `RegionId` column
+
+**Changed:**
+
+- `Column naming convention` attribute includes the following new provisions:
+  - Column IDs should not exceed 50 characters.
+- `EffectiveCost` column updated to clarify it does not mix or "blend" costs across multiple charges.
+- `Region` column renamed to `RegionName`.
+- `Tags` column updated to include:
+  - Tag keys that cannot have a value must use a boolean `true` as the tag value.
+
+**Fixed:**
+
+- `CommitmentDiscountType` column constraints were updated to show the column as required to align with the normative text.
+
+[All unreleased changes](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/v1.0-preview-cr...working_draft)
+
+<br>
+
+## v1.0-preview
+<sup>Announced November 15, 2023</sup>
+
+**Added:**
+
+- `Discount handling` attribute
+- `Key/value format` attribute
+- `Numeric format` attribute
+- `Unit format` attribute
+- `ChargeDescription` column
+- `ChargeFrequency` column
+- `ChargeSubcategory` column
+- `CommitmentDiscountCategory` column
+- `CommitmentDiscountId` column
+- `CommitmentDiscountName` column
+- `CommitmentDiscountType` column
+- `ListCost` column
+- `ListUnitPrice` column
+- `PricingCategory` column
+- `PricingQuantity` column
+- `PricingUnit` column
+- `ResourceType` column
+- `SkuId` column
+- `SkuPriceId` column
+- `Tags` column
+- `UsageQuantity` column
+- `UsageUnit` column
+
+**Changed:**
+
+- `Column naming convention` attribute includes the following new provisions:
+  - Columns with a `Category` suffix must be normalized.
+  - External, custom columns can be added but must be prefixed by `x_`.
+  - FOCUS columns must be listed before custom columns in the dataset.
+  - Added "SKU" as an exception to the "no acronyms" rule. The display name is "SKU" and column ID is `Sku`.
+- `AmortizedCost` column renamed to `EffectiveCost`
+- `ChargeType` column renamed to `ChargeCategory`
+
+[All 1.0-preview changes](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/v0.5-cr...v1.0-preview-cr)
+
+<br>
+
+## v0.5
```

Comment:
Originally, I wasn't planning to cover 0.5 but it felt strange to talk about GA relative to preview when preview didn't have a baseline to compare it to. This make the changelog complete for all announced versions.